06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,137 @@
+const books = require('./books.json');
+const books2 = require('./books2.json');
```

Comment:
PS: I don't see you testing the functions used to validate the files being uploaded. Let's fix please.